### PR TITLE
Add python sonarqube api recipe

### PR DESCRIPTION
**Pull request by JFreitas9**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/python-sonarqube-api/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  sha256: c254cd810ac05960864ba2077c912e15b8d35f990cda9fda9dbe20600adc4d4c
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (chrisburr) [inline recipes/python-sonarqube-api/meta.yaml:19]: The version constraint for Python in `host` and `run` should almost always match:
```suggestion
    - python >=2.7
```

Comment (JFreitas9) [reply recipes/python-sonarqube-api/meta.yaml]: Thank you.

**Comment by JFreitas9** [comment]
@chrisburr can you please review this PR? Thank you!

**Comment by JFreitas9** [comment]
@conda-forge/bot

**Comment by JFreitas9** [comment]
@chrisburr please check this PR. Thank you!

**Comment by JFreitas9** [comment]
@conda-forge/staged-recipes

**Comment by daniel88ferreira** [comment]
@conda-forge/help-python can you please review?

**Comment by synapticarbors** [comment]
I merged this since it looked good and has been sitting a while. Please note though that we'd generally like to move away from manually packaging the license file. I've opened a PR upstream to fix this (https://github.com/shijl0925/python-sonarqube-api/pull/14). If/when it's merged and a new release is made that includes the license file, please be sure to remove the manually packaged copy.

**Pull request merged**